src/main/java/com/smartcar/sdk/data/SmartcarResponse.java

```diff
@@ -110,6 +111,24 @@ public void setAge(Date age) {
     this.age = age;
   }
 
+  /**
```

Comment:
sidenote why the heck does this class have so many constructors :/

Reply:
yeah... I was looking to refactor this whole thing into a builder, but the constructors are referenced in enough places that would cause a much bigger change.